OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -82,8 +82,8 @@
   GCC:*_*_*_CC_FLAGS = -D DISABLE_NEW_DEPRECATED_INTERFACES
 
 [BuildOptions.X64]
-  GCC:*_*_*_CC_FLAGS = -fno-delete-null-pointer-checks -fwrapv
-  GCC:*_*_*_DLINK_FLAGS = -z noexecstack
+  GCC:*_GCC5_X64_CC_FLAGS = -fno-delete-null-pointer-checks -fwrapv -fstack-protector-strong -mstack-protector-guard=global -O0
```

Comment:
Remove -O0.

Reply:
In my case the fstack-protector-strong won't work without -O0 option.